sonar-php-plugin/src/test/java/org/sonar/plugins/php/phpunit/JUnitLogParserForPhpUnitTest.java

```diff
@@ -67,6 +67,6 @@ public void shouldThrowAnExceptionWhenReportDoesNotExist() throws Exception {
   @Test
   public void shouldParseComplexNestedSuites() throws Exception {
     final TestSuites suites = parser.parse(new File("src/test/resources/" + PhpTestUtils.PHPUNIT_REPORT_DIR + "phpunit-junit-report.xml"));
-    assertThat(suites.arrangeSuitesIntoTestFileReports().size()).isEqualTo(8);
+    assertThat(suites.arrangeSuitesIntoTestFileReports().size()).isEqualTo(9);
```

Comment:
I believe that all elements in the list returned by `arrangeSuitesIntoTestFileReports()` should have a different `file`. I believe that we still have to aggregate all instances of TestFileReport which refer to the same file. Otherwise, importing the report in SonarQube triggers an error: "Can not add the same measure twice on...". 

The description of [SONARPHP-927](https://jira.sonarsource.com/browse/SONARPHP-927) was definitely not helpful. I tried to clarify it in a comment.

Reply:
I should have testest end-to-end, sorry for that. Thanks for catching it! 
I've pushed a change that should fix the problem and achieve what you have posted in the comment. The assumption I made in the change is that if we have a "<testsuite .... file="abc">" element that all "<testcase>" that are nested in there (directly or indirectly) belong to that file. Let me know if I misunderstood something.